Sources/SRGDataProviderCombine/TVServices.swift

```diff
@@ -83,6 +83,16 @@ public extension SRGDataProvider {
         let request = requestTVScheduledLivestreams(for: vendor, signLanguageOnly: signLanguageOnly, eventType: eventType)
         return paginatedObjectsTriggeredPublisher(at: Page(request: request, size: pageSize), rootKey: "mediaList", type: SRGMedia.self, paginatedBy: signal)
     }
+
+    /**
+     *  List of TV livestreams.
+     *
+     *  - Parameter includeThirdPartyStreams: Whether third party livestreams should be included or not.
+     */
+    func audioLivestreams(for vendor: SRGVendor, includeThirdPartyStreams: Bool = true) -> AnyPublisher<[SRGMedia], Error> {
```

Comment:
TV or radio ?

if audio/radio, should be in `RadioServices.swift`.

It looks it's already implemented, not as a boolean but an enum `SRGContentProviders`:
https://github.com/SRGSSR/srgdataprovider-apple/blob/af9a6b300ec3ac5d49e00cf50ec5dacad3fc8458/Sources/SRGDataProviderCombine/RadioServices.swift#L74